nanoBitcoinSwitch.ino

```diff
@@ -47,9 +44,9 @@ void loop() {
     }
     oldBalance = balance;
     Serial.println("Paid");
-    digitalWrite(switchPin, HIGH);
+    digitalWrite(switchPin, LOW);
     delay(timePin);
-    digitalWrite(switchPin, LOW); 
+    digitalWrite(switchPin, HIGH);
```

Comment:
This still need to be LOW.
Always depending your HW but is prefered HIGH and then LOW